basket/news/tasks.py

```diff
@@ -241,6 +244,8 @@ def fxa_verified(data):
     # used to be handled by the fxa_register view
     email = data['email']
     fxa_id = data['uid']
+    create_date = data.get('createDate')
+    # TODO if create_date exists, convert it from ISO8601 to a datetime.
```

Comment:
How would you go about converting from an ISO8601 date (format `2018-05-18T13:21:54.188Z`) to a datetime? It looks like pyparsing is included, should that be used? Would you parse here or down in _update_fxa_info?

Reply:
I'd use `dateutil` which is also already installed. 

```python
from dateutil.parser import parse as parse_date
...
create_date = data.get('createDate')
if create_date:
    create_date = parse_date(create_date)
```